docs/index.md

```diff
@@ -0,0 +1,130 @@
+# Hopsworks Client
```

Comment:
This file is not required as it is auto-generated. https://github.com/logicalclocks/hopsworks-api/blob/dev-merge/auto_doc.py#L209

Reply:
Yes, the same is true for CONTRIBUTIONS.md in docs. Should I add both of them to gitignore?

Reply:
Sounds good